src/app/navbar/navbar.component.scss

```diff
@@ -0,0 +1,54 @@
+$header-background: #466bc1;
+$button-hover-color: #3859a5;
+$font-black: black;
+
+.profilepic{
+		height:100%;
+		width: 100%;
+		padding: 15px;
+
+}
+
+.sidebarheader{
+    background: $header-background;
+    padding: 4px;
+
+}
+
+.profile{
+		text-align: -webkit-center;
+		font-size: 20px;
+}
+
+.sidebar{
```

Comment:
Once the navbar is toggled in, it is like any other website. We can have a discussion upon it...